clang-tools-extra/clangd/ModuleFilesInfo.h

```diff
@@ -0,0 +1,118 @@
+//===----------------- ModuleFilesInfo.h -------------------------*- C++-*-===//
```

Comment:
This is a great class, but very misnamed! "Info" suggests this is a dumb data-carrier. But actually it owns the module files on disk, and the logic to build them!

I'd suggest `PrerequisiteModules`, which also suggests that the modules are the inputs for something in particular.

Reply:
Done by renaming. I don't quite understand the sentence "the modules are the inputs for something in particular.".